azurerm/internal/services/network/vpn_gateway_resource.go

```diff
@@ -78,6 +79,45 @@ func resourceArmVPNGateway() *schema.Resource {
 							Type:     schema.TypeString,
 							Computed: true,
 						},
+
+						"bgp_peering_addresses": {
```

Comment:
should this be 
```suggestion
						"bgp_peering_ip_addresses": {
```

or
```suggestion
						"ip_bgp_peering_addresses": {
						"instance_bgp_peering_addresses": {
```
given that ip_configuration_id seems to tie it into something that is not an ip resource id?

also the properties within could be
ip_???
custom_ips
default_ips
tunnel_ips

givenits within a block `bgp_peering_addresses` i think we can drop it from all the contained properties